collector/package.json

```diff
@@ -15,15 +15,13 @@
     "lint": "yarn prettier --ignore-path ../.prettierignore --write ./processSingleFile ./processLink ./utils index.js"
   },
   "dependencies": {
-    "@googleapis/youtube": "^9.0.0",
     "@xenova/transformers": "^2.11.0",
     "bcrypt": "^5.1.0",
     "body-parser": "^1.20.2",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "epub2": "^3.0.2",
     "express": "^4.18.2",
-    "extract-zip": "^2.0.1",
```

Comment:
is dependency of puppeteer, but is list as dep in lockfile, we dont use it directly.

Reply:
https://github.com/Mintplex-Labs/anything-llm/commit/719521c307d81c430d7d90872768d6de556c38d7